src/app/community/create/create.component.scss

```diff
@@ -0,0 +1,36 @@
+textarea.mat-input-element {
+  resize: none;
+  overflow: auto;
+}
+
+mat-card-content {
+  display: flex;
+  flex-direction: column;
+}
+
+.communityCreationForm {
+  margin: 30px;
+  height: 70vh;
```

Comment:
Do you think it still looks fine if we allow it to resize itself without any strict height?

![image](https://user-images.githubusercontent.com/11811793/37551704-2a337338-296b-11e8-95a5-60792caa1710.png)

Reply:
I personally find that looks kind of ugly... It also doesn't suggest that description can be expanded. The textarea is suppose to have this little tab on the bottom right that the user can use to expand the text area. But it causes it to go below the button which is why I removed that and gave a fixed height to signify the user that it's a text area not a single line of input.